highcharts-behavior.html

```diff
@@ -35,6 +35,7 @@
             yZoom: {type: Boolean, value: false, reflectToAttribute: true, observer: "_zoomUpdate"},
             loading: {type: Boolean, value: false, observer: "_loadingUpdate", reflectToAttribute:true},
             loadingMessage: {type: String, value: ""},
+            renderer: {type: Object, computed: "_renderer(_chart)"},
```

Comment:
I don't see why the `get` wouldn't work. The `'image' of undefined` does not seem to be an error on the Highcharts side, but rather your code accessing the render object before the component has initialized.

Could you confirm @kevinLB